src/dlsproto/client/UsageExamples.d

```diff
@@ -295,39 +309,18 @@ unittest
 unittest
 {
     // An example of handling a blocking Put request
-    struct BlockingPutExample
+    class BlockingPutExample : ExampleApp
     {
-        import ocean.io.Stdout;
-
-        DlsInit dls_init;
-
-        // Task-derivative which performs a Put request
-        static class PutTask : Task
-        {
-            private DlsClient dls;
-
-            this ( DlsClient dls )
-            {
-                this.dls = dls;
-            }
-
-            override public void run ( )
-            {
-                auto result = this.dls.blocking.put("channel", 0x1234,
-                    "value_to_put");
-                if ( !result.succeeded )
-                    Stdout.formatln("Put request failed");
-            }
-        }
-
-        // Method which initialises the client and starts a request
-        public void start ( )
+        protected override void example ( )
         {
-            dls_init.connectWithScheduler();
-
-            // Start a task
-            theScheduler.schedule(new PutTask(this.dls_init.dls));
-            theScheduler.eventLoop();
+            // Perform a blocking neo Put request and return a result struct
```

Comment:
Would be nice to have an example of the task-blocking Put with a proper notifier. (The one without the notifier isn't really recommended for use in applications.)

Reply:
(dhtproto has both, with a note about when you might want to use the simple one: https://github.com/sociomantic-tsunami/dhtproto/blob/neo/src/dhtproto/client/UsageExamples.d#L226-L292)

Reply:
Yes, I am going to provide more usage examples, but I wanted to do after this PR.

Reply:
Ok cool.